js/plugins.js

```diff
@@ -553,8 +553,47 @@ plugins.factory('userPlugins', function() {
         }
     });
 
+    /*
+     * TikTok embedded player
+     * Very similar to twitter
+     */
+    var tikTokPlugin = new UrlPlugin('TikTok', function(url) {
+        var regex = /^https?:\/\/(www\.)?tiktok\.com\/(.+)/i,
+            match = url.match(regex);
+
+        if (match) {
+
+            return function() {
+
+                var element = this.getElement();
+
+                GetEmbedContent( "https://www.tiktok.com/oembed?url=" + url, element);
+
+                function GetEmbedContent(embedurl, element) {
```

Comment:
Is there a reason that you turned this into a separate function that's used in exactly one place?

Reply:
Fixed, it had a reason but it did not play out because I could not get the vm.tiktok.com links to work.